lighthouse-cli/bin.ts

```diff
@@ -66,6 +67,8 @@ if (cliFlags.configPath) {
   config = require(cliFlags.configPath);
 } else if (cliFlags.perf) {
   config = perfOnlyConfig;
+} else if (cliFlags.a11y) {
```

Comment:
only downside here is that people might think you could do `lighthouse --perf --a11y` and get both, what if we supported that by nixing the config files and just appending each category that's enabled to `onlyCategories` of a default extending config

I can pickup that separately though